docs/general/remote-access.rst

```diff
@@ -178,18 +178,28 @@ into the image. However, there are two problems with this:
    format.
 
 To create dedicated host keys in the proper format, decide on a location, for example ``/etc/dropbear``, and create the
-new keys::
+new keys:
 
-  mkdir -p /etc/dropbear
-  for keytype in rsa ecdsa ed25519; do
-      dropbearkey -t "${keytype}" -f "/etc/dropbear/dropbear_${keytype}_host_key"
-  done
+.. tabs::
 
-.. note::
-  The dracut module expects to install RSA and ECDSA keys, so at minimum those keys should be created.
-  The mkinitcpio module supports RSA, ECDSA, and ED25519 keys.
+  .. group-tab:: Dracut
+
+      The dracut module expects to install RSA and ECDSA keys, so at minimum those keys should be created
+      We're making keys in PEM format because the dracut module insists on converting keys each time and doesn't accept keys in dropbear format::
+        
+        mkdir -p /etc/dropbear
+        for keytype in rsa ecdsa ed25519; do
+          ssh-keygen -g -N "" -m PEM -t "${keytype}" -f "/etc/dropbear/ssh_dracut_host_${keytype}_key"
```

Comment:
Leave the output files named as the later directions require, rather than changing those as well.
```suggestion
          ssh-keygen -g -N "" -m PEM -t "${keytype}" -f "/etc/dropbear/ssh_host_${keytype}_key"
```